react-frontend/vite.config.js

```diff
@@ -13,8 +13,8 @@ export default defineConfig(() => {
 		base: '/form/',
 		server: {
 			proxy: {
-				'/analyze': 'http://localhost:8000',
-				'/verify': 'http://localhost:8000'
+				'/analyze': 'http://localhost:80',
```

Comment:
Is the `:80` suffix necessary?

I don't know enough about this domain to evaluate the pros and cons of having the port explicitly specified if it turns out to be optional but am curious to hear your perspective.